### DISCUSSION TITLE
Exported game crashing, during loading of Terrain3D

### DISCUSSION THREAD

**Discussion by GeneralFletcher4** [TokisanGames/Terrain3D#438] [Questions] [answered]
Hi! I have created a game using this plugin, and it's wonderful. However when I try to export my game I'm facing a problem. The Godot Splash Screen boots up, then the game crashes.

![error](https://github.com/user-attachments/assets/776732d5-9fdb-42b3-9265-c96bdc480cde)

Looking at the logs the people who are trying to play my game get the following. I don't know what causes this, but I'd like to know :)

Any help would be appreciated

**Comment by TokisanGames** [comment]
4.3dev5 is not supported. Only 4.2 was supported, though support is starting with 4.3rc1. Upgrade and test again.

Also test exporting our demo.

What release version or build commit of the plugin? What platform? I need details to help you.

**Reply by GeneralFletcher4** [reply]
I changed engine version to 4.2.2 and it gives a similar error message. I tried using earlier version of 4.2, mainly 4.2.0, but the GD-extension threw errors. 

[error2](https://github.com/user-attachments/assets/310d0de3-951f-41eb-8844-71ebfbffa7fa)

Is 4.2.2 also supported or is it _only_ 4.3 rc1. The was in your answer makes me a bit unsure.

**Reply by TokisanGames** [reply, marked as answer]
Support had two meanings:
1) fully working and most tested. That includes 4.2.2 for the released binary, other 4.2.x for source builds. The released binary doesn't work pre-4.2.2. 
2) we will improve things for supported versions. I won't spend any time on 4.3dev* or 4.3beta*. Now that 4.3 is in rc, we will fix things.

I see your error with 4.2.2. Export with the demo and verify that works. If not we can look at that. It used to work. If so, then you have a working environment to compare what's different in your settings and project.

What platform are you on?

**Reply by TokisanGames** [reply]
Are you still having trouble with this issue? 4.2 has worked since released last year, in the editor and with exported versions.
4.3 rc* and stable versions have been working, though I have not tried exporting.
Still waiting on more information and testing. I'll close this in a few days if I can export the demo in 4.3 and there is no further response.

**Reply by TokisanGames** [reply]
I exported the demo with 4.3-stable and have no issues. There's been no further response, so closing this.
